vgpu/src/go.mod

```diff
@@ -1,6 +1,6 @@
 module vgpu-util
 
-go 1.20
+go 1.22.2
```

Comment:
Can we revert this instead? This indicate the MINIMUM go version required and does not affect the tooling choice.